packages/core/src/components/Link/Link.test.tsx

```diff
@@ -39,4 +39,31 @@ describe('<Link />', () => {
     });
     expect(getByText(testString)).toBeInTheDocument();
   });
+
+  describe('isExternalUri', () => {
+    it.each([
+      [true, 'http://'],
+      [true, 'https://'],
+      [true, 'https://some-host'],
+      [true, 'https://some-host/path#fragment'],
+      [true, 'https://some-host/path?param1=value'],
+      [true, 'slack://'],
+      [true, 'mailto://'],
```

Comment:
@andrewthauer Hmm, I think that this wrong. Mailto links should not continue with `//` but should just be `mailto:info@example.com`.

Reply:
Yeah mailto doesn't work 😆 we run into it

Reply:
Fixed in #4912